src/plugin.js

```diff
@@ -173,33 +175,34 @@ const onPlayerReady = (player, options) => {
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
-
     display = player.errorDisplay;
-
+    // The code snippet below is to make sure we dispose any child closeButtons before
+    // making the display closeable
+    if (display.getChild('closeButton')) {
+      display.removeChild('closeButton');
+    }
+    // Make the error display closeable, and we should get a close button
+    player.errorDisplay.closeable(true);
```

Comment:
Should probably replace `player.errorDisplay` with `display` here since we have that as a reference already.